packages/hardhat/contracts/Token.sol

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract Token
+///+interfaces
+{
+    // Allow SafeMath functions to be called for all uint256 types
+    using SafeMath for uint256; 
+
+    // Account used to deploy contract
+    address private contractOwner;         
+
+/*>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> ASSET VALUE TRACKING: TOKEN  <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<*/
+
+    string public name;
+    string public symbol;
+    uint256 public decimals;
+    
+    // Token balance for each address
+    mapping(address => uint256) balances;              
+
+    // Approval granted to transfer tokens by one address to another address                 
+    mapping (address => mapping (address => uint256)) internal allowed; 
+
+    // Tokens currently in circulation (you'll need to update this if you create more tokens)
+    uint256 public total;                  
+
+    // Tokens created when contract was deployed                             
+    uint256 public initialSupply;         
+
+    // Multiplier to convert to smallest unit                              
+    uint256 public UNIT_MULTIPLIER;                                     
+
+    constructor()  
+    {
+        contractOwner = msg.sender;       
+
+/*>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> ASSET VALUE TRACKING: TOKEN  <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<*/
+
+        name = "Token";             
+        symbol = "TKN";           
+        decimals = 18;     
+
+        // // Multiplier to convert to smallest unit
+        UNIT_MULTIPLIER = 10 ** uint256(decimals); 
+
+        uint256 supply = 1000;       
+        total = supply.mul(UNIT_MULTIPLIER);
+        initialSupply = total;
+
+        // // Assign entire initial supply to contract owner
+        balances[contractOwner] = total;    
+
+    }
+
+/*>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> ASSET VALUE TRACKING: TOKEN  <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<*/
+
+    // Fired when an account authorizes another account to spend tokens on its behalf
+
+    event Approval          
+                            (
+                                address indexed owner, 
+                                address indexed spender, 
+                                uint256 value
+                            );
+
+    // Fired when tokens are transferred from one account to another
+    event Transfer          
+                            (
+                                address indexed from, 
+                                address indexed to, 
+                                uint256 value
+                            );
+
+/*>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> ASSET VALUE TRACKING: TOKEN  <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<*/
```

Comment:
what are all these comment breaks for?

Reply:
Oh was going to use them to give more context on main sections throughout the contract. Will remove.